#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_thumb_filename-typo.dpatch by Yavor Doganov <yavor@gnu.org>
##
## DP: Typo fix.

@DPATCH@
diff -urNad kazehakase-0.5.8~/module/search/kz-hyper-estraier-search.c kazehakase-0.5.8/module/search/kz-hyper-estraier-search.c
--- kazehakase-0.5.8~/module/search/kz-hyper-estraier-search.c	2009-10-01 21:07:00.000000000 +0300
+++ kazehakase-0.5.8/module/search/kz-hyper-estraier-search.c	2009-10-03 19:15:24.000000000 +0300
@@ -401,7 +401,7 @@
 					       strlen(G_DIR_SEPARATOR_S));
 
 		thumb_filename =
-			egg_pixbuf_get_thumb_filename(uri,
+			egg_pixbuf_get_thumbnail_filename(uri,
 						      EGG_PIXBUF_THUMBNAIL_LARGE);
 		thumb_uri = g_strdup_printf("history-search:?image=%s",
 					    thumb_filename);
